impl_core/src/main/java/io/opencensus/implcore/trace/propagation/B3FormatImpl.java

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright 2017, OpenCensus Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.opencensus.implcore.trace.propagation;
+
+import static com.google.common.base.Preconditions.checkNotNull;
+
+import com.google.common.annotations.VisibleForTesting;
+import io.opencensus.trace.SpanContext;
+import io.opencensus.trace.SpanId;
+import io.opencensus.trace.TraceId;
+import io.opencensus.trace.TraceOptions;
+import io.opencensus.trace.propagation.B3Format;
+import java.text.ParseException;
+import java.util.EnumMap;
+import java.util.Map;
+
+/** Implementation of the {@link B3Format}. */
+@VisibleForTesting
+public final class B3FormatImpl extends B3Format {
+  // Used as the upper 8-bytes of the traceId when a 8-bytes traceId is received.
+  private static final String UPPER_TRACE_ID = "0000000000000000";
+  private static final String SAMPLED_VALUE = "1";
+  private static final String FLAGS_VALUE = "1";
+
+  @Override
+  public Map<HeaderName, String> toHeaders(SpanContext spanContext) {
```

Comment:
It looks like this implementation always ignore the parent span ID.  Since there isn't a one-to-one mapping between OpenCensus spans and B3 headers, I think there should be an explanation of the conversion in the API Javadocs.